Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -198,6 +199,19 @@ RefPtr<CSSValue> CSSPropertyParserWorkerSafe::parseFontFaceDisplay(const String&
     return parsedValue;
 }
 
+std::optional<FilterOperations> CSSPropertyParserWorkerSafe::parseFilterString(const Document& document, RenderStyle& style, const String& string, CSSParserMode mode)
```

Comment:
You can't pass Document or RenderStyle in here; those aren't worker-safe.

Reply:
I moved to this function to FilterOperationsBuilder.cpp to be a standalone function with the name `createFilterOperations()`.